composer-symfony5.4.json

```diff
@@ -84,7 +86,9 @@
             "App\\": "src/",
             "App\\Domain\\API\\": "api",
             "DoctrineMigrations\\": "migrations/src/",
-            "ServerManager\\": "ServerManager/classes/"
+            "ServerManager\\": "ServerManager/classes/",
+            "App\\Tests\\": "tests/",
+            "App\\DataFixtures\\": "fixtures/"
```

Comment:
otherwise these classes wouldn't be autoloaded in 'prod' (which is used by this new staging docker environment setup)